spec/lib/describe_indexer_spec.rb

```diff
@@ -107,9 +107,9 @@
         expect(indexed_record["migrated_bsi"]).to eq true
       end
 
-      xit 'referenced_by' do
-        expect(result['referenced_by_ssim'].first).to eq 'https://arxiv.org/abs/1903.06605'
-      end
+      # xit 'referenced_by' do
```

Comment:
This is not at all a criticism, but out of curiosity, why might you have chosen to remove this rather than remained in the pending state for `xit`?